releases/cloud/point-2024-05-09-release-note.md

```diff
@@ -0,0 +1,56 @@
+---
+description: This article lists improvements and bug fixes in the Syskit Point Cloud version 2024.3.48.11
+---
+
+# May 09, 2024
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business.
+
+## New Features
+
+* **New policy available:** Private Workspaces Shared with Everyone
+
+  * **Private Workspaces Shared with Everyone** are private workspaces that have given access to a large number of users or groups. 
+
+  * A violation is detected when:
+  
+    * **Access is given to dynamic security groups that are shared with a large number of people**: Everyone / Everyone except external users / All Users. 
+
+    * **A public workspace is changed to private** and a large number of people still have access or the workspace is still set to providing access to Everyone / Everyone except external users / All Users.
+
+  * The violation can be resolved either by the admin accepting the risk or by removing access to security groups shared with Everyone / Everyone except external users / All Users.
+
+  * [For more details about the Private Workspaces Shared with Everyone policy settings, read this article.](../../governance-and-automation/automated-workflows/private-workspaces-shared-with-everyone-admin.md)
+
+  * [To find out more on how to resolve the detected vulnerability, read the related Security & Compliance report article](../../governance-and-automation/security-compliance-checks/private-workspaces-shared-with-everyone.md)
+
+
+
+## Improvements & Bug Fixes
+
+* **Improvements made to Provisioning!**
+
+  * **When creating or editing provisioning templates** admins can now add security groups and mail-enabled security groups to the Additional Site Collection Admins section.
+
+  * **The SharePoint Site Provisioning template** now includes the option to Select a Language from the dropdown menu of available languages.
+  
+  * **Fixed an issue** that prevented all policies from being logged when applying policies on Provisioning templates. 
```

Comment:
where the provisioning template dialog didn't correctly display previously selected governance policies, ili tome slično